cloud/openssh/sshconfig.tmpl

```diff
@@ -1,10 +1,15 @@
 # Autogenerated by devbox. Do not modify manually.
 # ConfigVersion: {{ .ConfigVersion }}
 
-Host proxy.devbox.sh gateway.devbox.sh gateway.dev.devbox.sh
+Host proxy.devbox.sh
```

Comment:
is there a plan to add StrictHostKeyChecking to proxy as well?

Reply:
I think that we should. When we do was can reconsolidate these two host sections.